src/bricks/wallet/types.ts

```diff
@@ -87,7 +87,7 @@ interface IWalletBrickCheckoutCustomization {
    * Optional. Wallet Brick offers two optional customizable theme variables that receive hexadecimal values:
    * {elementsColor, headerColor}
    *
-   * @tutorial {@link https://www.mercadopago.com/developers/en/docs/checkout-bricks/wallet-brick/additional-customization/checkout-theme Checkout Theme customization} documentation.
+   * @tutorial {@link https://www.mercadopago.com.br/developers/pt/docs/checkout-pro/checkout-customization/user-interface/color-style Checkout Theme customization} documentation.
```

Comment:
Esse `.br` na URL vai ser mantido ou podemos remover? E o idioma, não deveria ser inglês?
```suggestion
   * @tutorial {@link https://www.mercadopago.com/developers/en/docs/checkout-pro/checkout-customization/user-interface/color-style Checkout Theme customization} documentation.
```